server/data/testutils/mockFeature.ts

```diff
@@ -0,0 +1,14 @@
+import config from '../../config'
+
+type FeatureNames = keyof typeof config.features
+
+// eslint-disable-next-line import/prefer-default-export
+export const setFeature = <Feature extends FeatureNames>(
```

Comment:
New test helper function to override values in `config.features`. Typing with generics means only keys (and their typed values) set in `config.ts` will work - and when these change it will cause TypeScript errors in tests to highlight that they need updating.